### DISCUSSION TITLE
1.0.0 release?

### DISCUSSION THREAD

**Discussion by osseonews** [vikejs/vike#1742] [Discussion] [answered]
Is there any ETA on a 1.0.0 release? We did a ton of work with this project but had to hold off using it in production because internally they wouldn't allow use of it until 1.0.0 version. I saw a post on some forum (don't remember where) where the lead on this project said it would reach 1.0.0 by end of summer 2024? Is that a viable timeline. thanks.

**Comment by brillout** [comment, marked as answer]
https://github.com/vikejs/vike/issues/985#issuecomment-2226838799.
